variables.tf

```
@@ -125,6 +125,12 @@ variable "service_account_email" {
   description = "The service account to run the function as."
 }
 
+variable "build_service_account" {
+  type        = string
+  default     = ""
```

As `google_cloudfunctions_function. build_service_account` is optional, let's use a default of `null`.

```suggestion
  default     = null
```

---

After making the change, please run `make docker_generate_docs` and include in your commit.